nipype/algorithms/confounds.py

```diff
@@ -585,7 +626,7 @@ def compute_dvars(in_file, in_mask, remove_zerovariance=False):
 
     if remove_zerovariance:
```

Comment:
We took a closer look on this (w. @chrisfilo) and I realized that you were 100% right about this. Sorry about that.

Since this ```remove_zerovariance``` input does not make any sense, let's get rid of it (including the ```zero_remove``` function that is more readable inline).

Again, my apologies for not looking at this more thoroughly before.